### DISCUSSION TITLE
how can I view the features selected in setup

### DISCUSSION THREAD

**Discussion by dheiver-santos** [pycaret/pycaret#1817] [General]
![Captura de tela 2021-11-07 223809](https://user-images.githubusercontent.com/85194452/140671365-99266844-8314-4cbd-a9e2-7aa6b61a2e16.png)

**Comment by ngupta23** [comment]
I think you can use get_config() and pass the right key. Please check to see if that works...

https://pycaret.readthedocs.io/en/latest/api/classification.html#pycaret.classification.get_config
